tests/bin/get-base-url.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable no-console, import/no-nodejs-modules */
+const path = require( 'path' );
+const fs = require( 'fs' );
+
+const testConfigFilePath = path.resolve( __dirname, '../e2e/config/default.json' );
+const testConfigFile = fs.readFileSync( testConfigFilePath );
+const testConfig = JSON.parse( testConfigFile );
+
+console.log( testConfig.url.slice( 0, -1 ) );
```

Comment:
do we still need this log?

Reply:
Yes, this is required, because this isn't actually a log, but an output that is read by the [bash script inside the `npm run docker:ping` command](https://github.com/Automattic/woocommerce-services/blob/dc6edb776bae802dfa56259d9a6cd1c2c12361c6/tests/bin/wait-for-e2e-build.sh#L11). This JS script reads the [JSON file used to initialise the WP Docker container](https://github.com/Automattic/woocommerce-services/blob/dc6edb776bae802dfa56259d9a6cd1c2c12361c6/tests/e2e/config/default.json#L2) and outputs the base URL, which should be http://localhost:8084. This value should be pretty constant, but I didn't want to hard-code it and needed to write this part in JS, because parsing JSON in bash is not a very fun time.